### PR TITLE
Implemented dynamic code of threshold updates page(#295eve6)

### PR DESCRIPTION
**Pull request by azkyakhan**
threshold-updtaes page

### CODE REVIEW

File: src/store/modules/job/actions.ts [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+      }
+    }).catch((err) => {
+      commit(types.JOB_RUNNING_UPDATED, { jobs: [], total: 0 });
+      console.error(err);
+      showToast(translate("Something went wrong"));
+    }) 
+  },
+
+  async fetchPendingJobs({ commit, dispatch, state }, payload){
+    console.log(payload)
```
Comment (adityasharma7) [inline src/store/modules/job/actions.ts:131]: It seems you missed a unwanted console log

File: src/views/ThresholdUpdates.vue [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+    async loadMorePendingJobs (event: any) {
+      this.getPendingJobs(
+        undefined,
+        Math.ceil(this.pendingJobs.length / (process.env.VUE_APP_VIEW_SIZE as any)).toString()
+      ).then(() => {
+        event.target.complete();
+      })
+    },
+    async refreshJobs(event: any) {
+        console.log(this.getCurrentEComStore.productStoreId)
```
Comment (adityasharma7) [inline src/views/ThresholdUpdates.vue:339]: Unwanted console log

File: src/views/ThresholdUpdates.vue [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+      this.isRetrying = true;
+      if(this.segmentSelected === 'pending') {
+          console.log(this.getCurrentEComStore.productStoreId)
```
Comment (adityasharma7) [inline src/views/ThresholdUpdates.vue:342]: Unwanted console log

File: src/views/ThresholdUpdates.vue [hunk 4]
```diff
@@ -0,0 +1,495 @@
...
+    //   this.freqType = id && (Object.entries(this.jobFrequencyType).find((freq) => freq[0] == id[0]) as any)[1]
+
+    //   if (this.currentJob && !this.isJobDetailAnimationCompleted) {
+    //     emitter.emit('playAnimation');
+    //     this.isJobDetailAnimationCompleted = true;
+    //   }
+    // },
+  },
+  created() {
+      console.log(this.getCurrentEComStore.productStoreId)
```
Comment (adityasharma7) [inline src/views/ThresholdUpdates.vue:441]: Unwanted console log

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/store/modules/job/actions.ts (hunk 1 above)
Comment (azkyakhan) [reply src/store/modules/job/actions.ts]: I have removed this unwanted console log

File: src/views/ThresholdUpdates.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/ThresholdUpdates.vue]: I have removed it sir.

File: src/views/ThresholdUpdates.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/ThresholdUpdates.vue]: I have removed it sir.

File: src/views/ThresholdUpdates.vue (hunk 4 above)
Comment (azkyakhan) [reply src/views/ThresholdUpdates.vue]: I have removed it sir.

**Comment by azkyakhan** [comment]
job-configuration

**Pull request merged**